Samples/excel-add-in-shape-dashboard/src/taskpane/taskpane.ts

```diff
@@ -0,0 +1,431 @@
+      ["CA-2017-104220", 42771, "Corporate", "Central", "Technology", "Phones", 207, "Iowa"],
+      ["CA-2017-110478", 42803, "Corporate", "West", "Office Supplies", "Envelopes", 15.25, "California"],
+      ["CA-2017-129567", 42815, "Consumer", "West", "Office Supplies", "Binders", 17.456, "California"],
+      ["CA-2017-144932", 42842, "Consumer", "East", "Office Supplies", "Art", 89.856, "Ohio"],
+      ["CA-2017-140963", 42899, "Home Office", "West", "Technology", "Phones", 279.96, "California"],
+      ["CA-2017-132934", 42912, "Consumer", "East", "Office Supplies", "Binders", 5.312, "New York"],
+      ["CA-2017-122105", 42914, "Consumer", "West", "Office Supplies", "Art", 95.92, "California"],
+      ["CA-2017-102946", 42921, "Home Office", "West", "Office Supplies", "Binders", 6.792, "Nevada"],
+      ["CA-2017-117947", 42970, "Corporate", "East", "Technology", "Phones", 37.91, "New York"],
+      ["CA-2017-137596", 42985, "Home Office", "Central", "Technology", "Phones", 1199.8, "Michigan"],
+      ["CA-2017-130043", 42997, "Corporate", "Central", "Office Supplies", "Paper", 31.872, "Texas"],
+      ["CA-2017-133333", 43000, "Corporate", "Central", "Office Supplies", "Paper", 22.72, "Wisconsin"],
+      ["CA-2017-126074", 43014, "Consumer", "Central", "Office Supplies", "Binders", 8.05, "Michigan"],
+      ["CA-2017-132976", 43025, "Corporate", "East", "Office Supplies", "Paper", 11.648, "Pennsylvania"],
+      ["CA-2017-132976", 43025, "Corporate", "East", "Office Supplies", "Labels", 24.84, "Pennsylvania"],
+      ["CA-2017-150707", 43027, "Home Office", "East", "Office Supplies", "Binders", 37.66, "Maryland"],
+      ["CA-2017-107727", 43031, "Home Office", "Central", "Office Supplies", "Paper", 29.472, "Texas"],
+      ["CA-2017-125388", 43031, "Corporate", "East", "Furniture", "Furnishings", 56.56, "Massachusetts"],
+      ["CA-2017-155558", 43041, "Consumer", "Central", "Office Supplies", "Labels", 6.16, "Minnesota"],
+      ["CA-2017-153339", 43044, "Corporate", "South", "Furniture", "Furnishings", 15.992, "Tennessee"],
+      ["US-2017-107272", 43051, "Consumer", "West", "Office Supplies", "Binders", 2.388, "Arizona"],
+      ["CA-2017-150959", 43052, "Consumer", "Central", "Office Supplies", "Labels", 10.44, "Texas"],
+      ["US-2017-110576", 43071, "Home Office", "East", "Furniture", "Furnishings", 36.488, "Pennsylvania"],
+      ["CA-2017-145233", 43074, "Consumer", "West", "Technology", "Phones", 470.376, "Colorado"],
+      ["CA-2017-117457", 43081, "Consumer", "West", "Technology", "Accessories", 179.95, "California"],
+      ["CA-2017-117457", 43081, "Consumer", "West", "Office Supplies", "Paper", 27.15, "California"],
+      ["US-2017-145366", 43082, "Corporate", "East", "Office Supplies", "Storage", 37.208, "Ohio"],
+      ["CA-2017-155376", 43096, "Consumer", "Central", "Office Supplies", "Appliances", 839.43, "Missouri"],
+    ]);
+
+    // Prepare table data for RegionMap chart.
```

Comment:
Unless I'm missing something, there's not a region map chart in this sample. Double-check that this code is needed.

Reply:
It's used in the dashboard info.